### DISCUSSION TITLE
User to asset details

### DISCUSSION THREAD

**Discussion by ninja5244** [CrowdStrike/psfalcon#405] [Q&A]
Is there any way to retrieve user to asset details from CS IDP?

**Comment by bk-cs** [comment]
If it comes from Identity Protection, the data will need to be gathered using a GraphQL search and [Invoke-FalconIdentityGraph](https://github.com/crowdstrike/psfalcon/wiki/Invoke-FalconIdentityGraph).

Unfortunately I don't know what GraphQL query you'd need.
